src/Servers/HttpSys/src/AuthenticationManager.cs

```diff
@@ -63,6 +64,25 @@ public bool AllowAnonymous
     /// </summary>
     public string? AuthenticationDisplayName { get; set; }
 
+    /// <summary>
+    /// If true, the Kerberos authentication credentials are persisted per connection
+    /// and re-used for subsequent anonymous requests on the same connection.
+    /// Kerberos or Negotiate authentication must be enabled. The default is false.
+    /// This option maps to the native HTTP_AUTH_EX_FLAG_ENABLE_KERBEROS_CREDENTIAL_CACHING flag.
+    /// <see href="https://learn.microsoft.com/en-us/windows/win32/api/http/ns-http-http_server_authentication_info"/>
+    /// </summary>
+    public bool EnableKerberosCredentialCaching { get; set; }
+
+    /// <summary>
+    /// If true, the server captures the current caller's credentials and impersonates
```

Comment:
This isn't my area of expertise, but I understood from @halter73's [comment](https://github.com/dotnet/aspnetcore/issues/51990#issuecomment-1832617112) that "current" was a confusing description of which credentials would be captured and that we should say something about using the host's credentials.  It seems like maybe you've said that in the next sentence, but then I'm not sure how to reconcile it with the first.  Maybe they could be combined?

> If true, credentials are captured from the security context of the thread that performs host initialization and used during Kerberos or Negotiate authentication.

Again, I don't actually know what that sentence means, I'm just comparing the text of multiple explanations. :wink:

Reply:
I like it!  Thanks for the explanation!

Reply:
Committed in https://github.com/dotnet/aspnetcore/commit/a2885d991de74bce088652f5711dd1e8b0116ec8, thanks!